src/NServiceBus.Core/obsoletes.cs

```diff
@@ -336,6 +337,17 @@ public static void RegisterEncryptionService(this EndpointConfiguration config,
             throw new NotImplementedException();
         }
     }
+
+    [ObsoleteEx(
+        Message = "Implement `IWantToRunWhenEndpointStartsAndStops` in the NServiceBus.Host package or NServiceBus.Host.AzureCloudService package instead", 
```

Comment:
should we also recommend to consider the upgrade guide to see what to use in case they're not using the host?